pom.xml

```diff
@@ -6,7 +6,7 @@
         <artifactId>spring-boot-starter-parent</artifactId>
```

Comment:
did you run some sort of clean-up or styling on this file. Seems to have been moved about alot